index.html

```diff
@@ -352,7 +366,7 @@ <h3>
 							<p class="milestone"><b>Expected completion:</b> Q1 2023</p>
 						</dd>
 
-						<dt id="a11y" class="spec">EPUB Accessibility</dt>
+						<dt id="a11y" class="spec">EPUB Accessibility 1.X</dt>
 						<dd>
               				<p>
 								The next version of the EPUB Accessibility defines discovery and content accessibility requirements for EPUB Publications. The specifications is build on <a href='https://www.w3.org/TR/WCAG22/'>WCAG 2.x</a> and adds accessibility requirements specific to publishing.
```

Comment:
better to say "EPUB Accessibility defines discovery and content accessibility requirements for EPUB Publications." instead of "The next version of the EPUB Accessibility defines discovery and content accessibility requirements for EPUB Publications." The existing version of EPUB accessibility defines discovery and content accessibility requirement, it is not something that will happen in next version, it is already there. It will also be aligned to definition of other EPUB family of specs, we are defining what specs are and not saying what will be in the next version. e.g. definition of EPUB Open Container Format 3.X says This specification defines a file format and processing model for encapsulating the set of related resources that comprise an EPUB Publication into a single-file container

Reply:
Thanks @avneeshsingh, the commit https://github.com/w3c/epub-3-wg-charter/pull/47/commits/6f385697ac5e147a8d022f0d63c28345acd65520 takes care of those changes.